### ISSUE TITLE
Credential with new password

### ISSUE THREAD

**Issue by Teutenberg** [MicrosoftDocs/PowerShell-Docs#3718]
If the user credential is regenerated with a new password, the service resource doesn't update the password for the service. It will test in the desired state, but the service will fail on next restart. My current work around is to set the service to use a built in account, then reset it back to the original account with updated password. 

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: fd6170c3-833e-6483-a75c-ecb5e3612115
* Version Independent ID: 19d40960-8d93-9f19-9f06-44f6d5ea742f
* Content: [DSC Service Resource](https://docs.microsoft.com/en-us/powershell/dsc/reference/resources/windows/serviceresource)
* Content Source: [dsc/reference/resources/windows/serviceResource.md](https://github.com/MicrosoftDocs/PowerShell-Docs/blob/staging/dsc/reference/resources/windows/serviceResource.md)
* Product: **powershell**
* GitHub Login: @bobbytreed
* Microsoft Alias: **robreed**

**Comment by bobbytreed** [comment]
Can you elaborate more on this? Do you have an example?

**Comment by Teutenberg** [comment]
I was testing a process to add a local windows user as the service account login. 
I was using dsc to create and assign a random guid password to the local user account. I would assign a new password each time I ran the process. 

I would find the process worked the first time, but subsequent runs the service would fail to restart. 
I found that the service dsc would test in the desired state even though I changed the users password. 

My work around was to change the service account back to a built in account, then use dsc to set the service user login again.  

Sorry, I didn't keep any example code. I was using Invoke-DscResource to test.
